analysis/mailchimp/mailchimp_gains_losses.sql

```diff
@@ -0,0 +1,47 @@
+with
+email_summary as
+(
+	select *
+	from {{env.schema}}.mailchimp_email_summary
+),
+
+gains as (
+	select date_trunc('month', signup_date) as month, count(*) total_gains
+	from {{env.schema}}.mailchimp_members
+	group by date_trunc('month', signup_date)
+),
+
+losses as (
+	select month, count(*) as total_losses
+	from
+	(
+		-- find everyone who unsubscribed
+	  	select date_trunc('month', unsubscribed_date) as month
+	  	from email_summary
+	  	where unsubscribed_date is not null
+	  	-- add everyone with a hard bounce
+	  	union
```

Comment:
@jthandy Yes, of course! This was written with email IDs first. Updated in the PR.